agent/src/main.cpp

```diff
@@ -137,8 +138,8 @@ int main(int argc, char **argv)
     int userID = 0;
     bool keepContainersAlive = false;
     int timeout = -2;
-    Glib::ustring serviceManifestDir = SERVICE_MANIFEST_DIR_OPT;
-    Glib::ustring defaultServiceManifestDir = DEFAULT_SERVICE_MANIFEST_DIR_OPT;
+    Glib::ustring serviceManifestDir = "";
+    Glib::ustring defaultServiceManifestDir = "";
```

Comment:
With the config parser we're using, that is not currently possible, unless we write our own callbacks (which feels a bit overkill imo)